lib/modules/marker.rb

```diff
@@ -0,0 +1,29 @@
+module Codebreaker
+  module Marker
+    def check_win
+      if win?
```

Comment:
No, I can not. This logic assumed 3 aspects of game: win, game over, marking. It maybe possible if I using nested 'if', but I think that 'elsif' is better.